src/EditorFeatures/Core/FindUsages/FindUsagesContext.cs

```diff
@@ -13,8 +14,11 @@ internal abstract class FindUsagesContext : IFindUsagesContext
     {
         public virtual CancellationToken CancellationToken { get; }
 
+        public IStreamingProgressTracker ProgressTracker { get; }
+
         protected FindUsagesContext()
         {
+            this.ProgressTracker = new StreamingProgressTracker(this.ReportProgressAsync);
```

Comment:
this is what all the clients used to do. we now push this into the context-object itself.  Clients just use the ProgressTracker and it aggregates all the values and pushes the right pairs to ReportProgressAsync.

Reply:
So the semantics for anybody still directly calling this is it just overwrites any streaming-reported progress? I guess if this is being left around for F# that's fine because generally people are using one or the other, but not both?